config/env/private-public.json

```diff
@@ -3,18 +3,18 @@
     "services": {
       "ceramic": {
         "mode": "client",
-        "apiURL": "https://ceramic-private-dev.3boxlabs.com"
+        "apiURL": "https://ceramic-private-qa.3boxlabs.com"
       },
       "ipfs": {
         "mode": "none"
       },
       "ceramicClient": {
         "mode": "client",
-        "apiURL": "https://ceramic-dev.3boxlabs.com"
+        "apiURL": "https://ceramic-qa.3boxlabs.com"
       }
     }
   },
   "serviceLauncher": {
     "services": []
   }
-}
\ No newline at end of file
+}
```

Comment:
Can revert this

Reply:
why should we revert this?

Reply:
Oh wait, sorry. I misread the change. Looks good.